File: topics/storing-using-data/examples/TimesTable.pas
// 
// Program: TimesTable.pas
// Displays the Times Table from 1 x n to 10 x n.
//
program TimesTable;

procedure Main();
var
    number: Integer;
begin
    WriteLn('Times Table');
    
    Write('Enter number: ');
    ReadLn(number);
    
    WriteLn('-----------------');
    WriteLn(' 1 x ', number, ' = ', 1 * number);
    WriteLn(' 2 x ', number, ' = ', 2 * number);
    WriteLn(' 3 x ', number, ' = ', 3 * number);
    WriteLn(' 4 x ', number, ' = ', 4 * number);
    WriteLn(' 5 x ', number, ' = ', 5 * number);
    WriteLn(' 6 x ', number, ' = ', 6 * number);
    WriteLn(' 7 x ', number, ' = ', 7 * number);
    WriteLn(' 8 x ', number, ' = ', 8 * number);
    WriteLn(' 9 x ', number, ' = ', 9 * number);
    WriteLn('10 x ', number, ' = ', 10 * number);
    WriteLn('-----------------');
end;

begin
    Main();
end.

File: topics/storing-using-data/examples/times_table.c
/*
* Program: times_table.c
* Displays the Times Table from 1 x n to 10 x n.
*/

#include <stdio.h>

int main()
{
    int number = 0;
    
    printf("Times Table\n");
    
    printf("Enter number: ");
    scanf("%d", &number);
    
    printf("-----------------\n");
    printf(" 1 x %d = %d\n", number, 1 * number);
    printf(" 2 x %d = %d\n", number, 2 * number);
    printf(" 3 x %d = %d\n", number, 3 * number);
    printf(" 4 x %d = %d\n", number, 4 * number);
    printf(" 5 x %d = %d\n", number, 5 * number);
    printf(" 6 x %d = %d\n", number, 6 * number);
    printf(" 7 x %d = %d\n", number, 7 * number);
    printf(" 8 x %d = %d\n", number, 8 * number);
    printf(" 9 x %d = %d\n", number, 9 * number);
    printf("10 x %d = %d\n", number, 10 * number);
    printf("-----------------\n");
    
    return 0;
}